### PR TITLE
LPS-157151 Testray Automated | master

### PR DESCRIPTION
**Pull request by joelrocha68**
https://issues.liferay.com/browse/LPS-157151

### CODE REVIEW

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:stop

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:stop

**Comment by joelrocha68** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayFiltersCasesAdministrator.testcase [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+	setUp {
+		task ("Login and create a Testray Site") {
+			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+
+			Testray.addTestraySite(siteName = ${testSiteName});
+		}
+
+		task ("Given that I create all the necessary objects") {
```
Comment (nikki-pru) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayFiltersCasesAdministrator.testcase:26]: Please list what objects are created. Ex:
```task ("Setup: Given that I create a Project, Team, Component, and Test Cases")```

**Review by nikki-pru — COMMENTED** [review]
@joelrocha68 , Small change in the Setup please

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:site-initializer-testray:nocompile

**Pull request closed without merging**